### PR TITLE
Add backend connector service

### PR DESCRIPTION
**Pull request by csehlke**
add backend connecter to our frontend

### CODE REVIEW

**Comment by csehlke** [comment]
- [ ]  before merging the backend should be ready

**Review by jyotirmay123 — APPROVED** [review]
Looks good.

File: UI/src/app/home/home.component.html [hunk 1]
```diff
@@ -29,6 +29,6 @@
   </div>
 </div>
 <app-chart #chart
-           [layout]="layout"
+           [layout]="options"
```
Comment (csehlke) [inline UI/src/app/home/home.component.html]: needs fix :)

Comment (csehlke) [reply UI/src/app/home/home.component.html]: fixed

**Pull request merged**